Project4.md

```diff
@@ -178,21 +178,23 @@ Django supplies well tested code for hashing and checking
 passwords. You should use this code rather than trying to create your
 own versions. Use `django.contrib.auth.hashers` module's
 `make_password` and `check_password`. My sameple
-Project 2 code shows an example of using `make_password`.
+Project 2 code (https://github.com/thomaspinckney3/stuff-models/blob/master/stuff/main.py) shows an example of using `make_password`. Read more at the Django Docs https://docs.djangoproject.com/en/1.10/topics/auth/passwords/#module-django.contrib.auth.hashers
```

Comment:
We told students to use Django 1.9 so we should point them to that version of the docs.